datalad/interface/__init__.py

```diff
@@ -12,57 +12,41 @@
 
 __docformat__ = 'restructuredtext'
 
-# the following should be series of imports of interface implementations
-# the shall be exposed in the Python API and the cmdline interface
-#from .sparql_query import SPARQLQuery
-from datalad.distribution.modify_subhandle_urls import ModifySubhandleURLs
-from datalad.distribution.move import Move
-from datalad.distribution.uninstall import Uninstall
-from datalad.distribution.update import Update
-from .add_archive_content import AddArchiveContent
-from .clean import Clean
-from .crawl import Crawl
-from .download_url import DownloadURL
-from .ls import Ls
-from .pull import Pull
-from .push import Push
-from .test import Test
-from ..distribution.add_sibling import AddSibling
-from ..distribution.create_publication_target_sshwebserver import \
-    CreatePublicationTargetSSHWebserver
-from ..distribution.install import Install
-from ..distribution.publish import Publish
-
-# very optional ones
-from ..distribution.tests.create_test_dataset import CreateTestDataset
 
+# the following should be series of import definitions for interface implementations
+# that shall be exposed in the Python API and the cmdline interface
 # all interfaces should be associated with (at least) one of the groups below
 _group_dataset = (
     'Commands for dataset operations',
     [
-        Install,
-        Publish,
-        Uninstall,
-        Move,
-        Update,
-        CreatePublicationTargetSSHWebserver,
-        AddSibling,
-        ModifySubhandleURLs,
+        # source module, source object[, dest. cmdline name[, dest python name]]
+        # src module can be relative, but has to be relative to the main 'datalad' package
+        ('datalad.distribution.install', 'Install'),
```

Comment:
I guess if automating we could even drop specifying which class and just go through all attributes of the imported module and pick up "supported" ones.  I don't think that runtime impact would be any noteable, while allowing for flexibility to support multiple things without duplicating names here etc